manifests/dev.pp

```diff
@@ -33,7 +33,7 @@
 
   # Default PHP come with xml module and no seperate package for it
   if $facts['os']['name'] == 'Ubuntu' {
-    ensure_packages(["${php::package_prefix}xml"], { ensure  => present, require => $require, })
+    stdlib::ensure_packages(["${php::package_prefix}xml"], { ensure  => present, require => $require, })
```

Comment:
you have to bump the `puppetlabs/stdlib` requirement to `>= 9` for that

Reply:
and given https://github.com/voxpupuli/puppet-php/pull/691 did not end up red when the upper bound was bumped, I guess this line has no test coverage?